priv/repo/migrations/20170126124818_create_user.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Healthlocker.Repo.Migrations.CreateUser do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :name, :string
+      add :dob, :date
+      add :email, :string
+      add :password, :string
+      add :pin, :string
+      add :securityquestion, :string
```

Comment:
Yep, I will change that up. I had to change it to that in the html so it will display nicer, so makes sense.